tests/covalent_tests/dispatcher_plugins/local_test.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 Agnostiq Inc.
+#
+# This file is part of Covalent.
+#
+# Licensed under the GNU Affero General Public License 3.0 (the "License").
+# A copy of the License may be obtained with this software package or at
+#
+#      https://www.gnu.org/licenses/agpl-3.0.en.html
+#
+# Use of this file is prohibited except in compliance with the License. Any
+# modifications or derivative works of this file must retain this copyright
+# notice, and modified files must contain a notice indicating that they have
+# been altered from the originals.
+#
+# Covalent is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the License for more details.
+#
+# Relief from the License may be granted by purchasing a commercial license.
+
+"""Unit tests for the Dispatcher."""
+
+import covalent as ct
+
+
+def test_dispatch_workflow_on_stop():
+    @ct.electron
+    def join_words(a, b):
+        return ", ".join([a, b])
+
+    @ct.electron
+    def excitement(a):
+        return f"{a}!"
+
+    # Construct a workflow of tasks
+    @ct.lattice
+    def simple_workflow(a, b):
+        phrase = join_words(a, b)
+        return excitement(phrase)
+
+    # Dispatch the workflow
+    dispatch_id = ct.dispatch(simple_workflow)("Hello", "World")
+    assert (
+        dispatch_id
+        == 'Covalent server is not running. Start the server using "covalent start" and try dispatching workflows.'
```

Comment:
I don't know if it should be returning the dispatch id set to a string explaining the server is not running. Seems like it would be more appropriate to raise an error.

Reply:
@FyzHsn I have made the changes regarding the string set which is assigned to the dispatch id.

Reply:
@Aravind-psiog I agree with @FyzHsn such that there should be an error raised when calling `ct.dispatch` when the server is running. Currently there is still a string being returned when it should be an exception raised.